autoload/ale/fixers/yapf.vim

```diff
@@ -1,13 +1,18 @@
 " Author: w0rp <devw0rp@gmail.com>
 " Description: Fixing Python files with yapf.
 
-function! ale#fixers#yapf#Fix(buffer) abort
+function! ale#fixers#yapf#Fix(buffer, lines) abort
```

Comment:
You can remove the `lines` argument here and return `0` instead. You can return `0` to skip any fixer.